README.md

```diff
@@ -1,24 +1,38 @@
-# README
+# How are you doing? ((Backend))
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Table of contents
 
-Things you may want to cover:
+- [Schema](#Schema)
+- [Endpoints](#endpoints)
+- [Contributors](#contributors)
 
-* Ruby version
+## Schema: 
 
-* System dependencies
+<img src="./app/assets/images/schema.png" alt="The schema of the project includes 4 tables."  width="600" height="350" />
 
-* Configuration
 
-* Database creation
+## Setup
 
-* Database initialization
+- `Ruby 2.7.4`
+- `Rails 5.2.8.1'`
+- [Fork this repository](https://github.com/Alaina-Noel/How_Are_You_Doing_BE)
+- Clone your fork
+- From the command line, install gems and set up your DB:
+- `bundle install`
+- `rails db:create`
+- `rails db:migrate`
+- Install Figaro with `bundle exec figaro install` to create an `application.yml` file locally (this to be updated with any needed ENV variables!!!)
```

Comment:
We don't have Figaro at this point in the BE Gemfile do we want to hold off on this part of the setup in the ReadMe until we are using it?